doom-themes-base.el

```diff
@@ -816,6 +816,8 @@
     (ivy-posframe-border :inherit 'internal-border)
     ;;;; selectrum
     (selectrum-current-candidate :background region :extend t)
+    ;;;; embark
+    (embark-target :background region :extend t)
```

Comment:
Could you move this up above `;;;; enh-ruby-mode ...` and below `;;;; elscreen` (i.e. [above this line](https://github.com/doomemacs/themes/blob/b5ff201f4bea4286e9ed015a2043cf2394182232/doom-themes-base.el#L516))? These sections are ordered alphabetically.

Reply:
Also, use hard inheritance here:

```lisp
((embark-target &inherit vertico-current))
```

The difference between this and `:inherit` is that `&inherit` copies the inherited face's attributes when the theme is compiled, rather than loaded, and allows it to work even if vertico is never loaded/installed.

Reply:
Thanks for the feedback! I've incorporated all the suggested changes.